cnf-tests/testsuites/e2esuite/multinetworkpolicy/multinetworkpolicy_sriov.go

```diff
@@ -0,0 +1,750 @@
+package multinetworkpolicy
+
+import (
+	"context"
+	"reflect"
+	"time"
+
+	. "github.com/onsi/ginkgo" //lint:ignore ST1001 used for tests
+	. "github.com/onsi/gomega" //lint:ignore ST1001 used for tests
+
+	multinetpolicyv1 "github.com/k8snetworkplumbingwg/multi-networkpolicy/pkg/apis/k8s.cni.cncf.io/v1beta1"
+	netattdefv1 "github.com/k8snetworkplumbingwg/network-attachment-definition-client/pkg/apis/k8s.cni.cncf.io/v1"
+	sriovClean "github.com/k8snetworkplumbingwg/sriov-network-operator/test/util/clean"
+	sriovtestclient "github.com/k8snetworkplumbingwg/sriov-network-operator/test/util/client"
+	sriovcluster "github.com/k8snetworkplumbingwg/sriov-network-operator/test/util/cluster"
+	sriovNamespaces "github.com/k8snetworkplumbingwg/sriov-network-operator/test/util/namespaces"
+	sriovNetwork "github.com/k8snetworkplumbingwg/sriov-network-operator/test/util/network"
+	client "github.com/openshift-kni/cnf-features-deploy/cnf-tests/testsuites/pkg/client"
+	"github.com/openshift-kni/cnf-features-deploy/cnf-tests/testsuites/pkg/discovery"
+	"github.com/openshift-kni/cnf-features-deploy/cnf-tests/testsuites/pkg/execute"
+	"github.com/openshift-kni/cnf-features-deploy/cnf-tests/testsuites/pkg/images"
+	"github.com/openshift-kni/cnf-features-deploy/cnf-tests/testsuites/pkg/namespaces"
+	"github.com/openshift-kni/cnf-features-deploy/cnf-tests/testsuites/pkg/nodes"
+	"github.com/openshift-kni/cnf-features-deploy/cnf-tests/testsuites/pkg/pods"
+	"github.com/openshift-kni/cnf-features-deploy/cnf-tests/testsuites/pkg/sriov"
+	"github.com/openshift-kni/cnf-features-deploy/cnf-tests/testsuites/pkg/utils"
+	corev1 "k8s.io/api/core/v1"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+
+	runtimeclient "sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const (
+	SRIOV_RESOURCE    = "sriovnicMultiNetworkpolicyResource"
+	TEST_NETWORK_NAME = "test-multi-networkpolicy-sriov-network"
+	DEFAULT_NAMESPACE = "default"
+	TEST_NETWORK      = DEFAULT_NAMESPACE + "/" + TEST_NETWORK_NAME
+)
+
+// The following test model is common to all tests in the [multinetworkpolicy] test suite:
+// Three namespaces nsX, nsY, nsZ, each of them with three pods podA, podB, podC
+const nsX string = utils.MultiNetworkPolicyNamespaceX
+const nsY string = utils.MultiNetworkPolicyNamespaceY
+const nsZ string = utils.MultiNetworkPolicyNamespaceZ
+
+var nsX_podA, nsX_podB, nsX_podC *corev1.Pod
+var nsY_podA, nsY_podB, nsY_podC *corev1.Pod
+var nsZ_podA, nsZ_podB, nsZ_podC *corev1.Pod
+
+var port5555 intstr.IntOrString = intstr.FromInt(5555)
+var port6666 intstr.IntOrString = intstr.FromInt(6666)
+var protoTCP corev1.Protocol = corev1.ProtocolTCP
+var protoUDP corev1.Protocol = corev1.ProtocolUDP
+
+var _ = Describe("[sriov] MultiNetworkPolicy integration", func() {
```

Comment:
That's true: If I understood correctly gingko's focus parameter is calculated from FEATURES list in an OR fashion:

```
# hack/run-functest.sh

FOCUS="-ginkgo.focus="$(echo "$FEATURES" | tr ' ' '|')
```

If I add `[multinetworkpolicy]` to the description, it would run even if sriov feature is not selected. Would it be ok?

it seems to be a short blanked problem